cmd/kyma/alpha/create/module/module.go

```diff
@@ -54,12 +53,15 @@ Build module modB in version 3.2.1 and push it to a local registry "unsigned" su
 		RunE:    func(_ *cobra.Command, args []string) error { return c.Run(args) },
 		Aliases: []string{"mod"},
 	}
-	cmd.Args = cobra.ExactArgs(3)
 
+	cmd.Flags().StringVar(&o.Version, "version", "", "Version of the module. This flag is mandatory.")
+	cmd.Flags().StringVarP(&o.Name, "name", "n", "", "Override the module name of the kubebuilder project. If the module is not a kubebuilder project this flag is mandatory.")
+	cmd.Flags().StringVarP(&o.Path, "path", "p", "", "Path to the module contents. (default current directory)")
+	cmd.Flags().StringVar(&o.ModPath, "mod-path", "./mod", "Specifies the target path where the module will be built into. If the path already has a descriptor use the overwrite flag to overwrite it. This flag shuld only be used by experienced users.")
```

Comment:
```suggestion
	cmd.Flags().StringVar(&o.ModPath, "mod-path", "./mod", "Specifies the target path where the module will be built. If the path already has a descriptor use the overwrite flag to overwrite it. This flag shuld only be used by experienced users.")
```

Reply:
Chnged the flag name and description so this suggestion does not apply anymore